package.json

```diff
@@ -49,7 +49,7 @@
     "react-dom-factories": "^1.0.0"
   },
   "peerDependencies": {
-    "react": "^0.14.9 || ^15.3.0 || ^16.0.0"
+    "react": "^0.14.9 || ^15.3.0 || ^16.0.0-beta"
```

Comment:
Agree, even if it is technically true, it doesn't make sense for package consumers to mark compatibility with unreleased versions with unstable APIs.